blogposts/2021/dev-tool-time-leah-culver.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "'If you're still using Google Docs, what are you doing?!' Dev Tool Time with Leah Culver"
```

Comment:
While I thought this comment was funny, from a dev perspective it's better to not lead with google docs as it seems not as dev-focused. This comment also comes across as negative rather than positive. Could we pick something else?